.github/workflows/release_notifications.yml

```diff
@@ -0,0 +1,178 @@
+# description of this workflow, can be anything you want
+name: Send Release Notifcations
+
+# we need to let GitHub know _when_ we want to release, typically only when we create a new tag.
+# this will target only tags, and not all pushes to the master branch.
+# this part can be heavily customized to your liking, like targeting only tags that match a certain word,
+# other branches or even pullrequests.
+# 'workflow_dispatch' allows us to re-run a failed job.
+on:
+  workflow_dispatch:
+  release:
+    types: [published]
+
+# a workflow is built up as jobs, and within these jobs are steps
+jobs:
+
+## Note: this is slightly messy, there might be a better way to do this, but im not sure how.
+## There are 2 options, either send all notifications as part of a single 'job'
+## or send each notification as seperate jobs.
+## The benefit of seperate jobs is that we can re-run specific social medias compeltely seperately from each other
+## this allows us to just kick off a specific notification without triggering the others.
+## the downside is mainly configuration bloat.  There are a lot of repeated tasks here
+## (such as clone, generate changelog, generate twitter text, save the info)
+## maybe someone smarter than me can help me understand how to save the changelog and twitter text once into a ENV var and use it across differnet jobs.
+
+############### DISCORD ####################
+  # "release-notifcations" is a job, you can name it anything you want
+  discord-release-notifcations:
+    # we can run our steps on pretty much anything, but the "ubuntu-latest" image is a safe bet
+    runs-on: ubuntu-latest
+
+    # specify the environment variables used by the packager, matching the secrets from the project on GitHub
+    env:
+      MESSAGE: "New WeakAuras Release"
+
+    # "steps" holds a list of all the steps needed to package and release our AddOn
+    steps:
+      # we first have to clone the AddOn project, this is a required step
+      - name: Clone project
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0  # gets git history for changelogs
+
+      - name: Generate Changelog
+        id: Changelog
+        run: ./generate_changelog.sh
+
+      - name: save changelog in variable
```

Comment:
Someone smarter then me aka @Stanzilla can prob come up with a better way to reroute the output of the shell script into a variable

Reply:
yea to be honest there was an warning it was thawing me about using this stuff, but i looked up the news post they made and it was in 2022 and still haven't deprecated it because so many people use it.